arbisoft/arbisoft/spiders/jobs_spider.py

```diff
@@ -0,0 +1,75 @@
+import scrapy
+import datetime
+from ..items import JobItem
+import re
+
+
+class JobsSpider(scrapy.Spider):
+    name = 'jobs'
+    start_urls = [
+        'https://news.ycombinator.com/jobs'
+    ]
+
+    def parse(self, response):
+
+        items = response.css("tr.athing td.title, td.subtext")
+        job_titles = []
+        company_urls = []
+        job_urls = []
+        job_posting_dates = []
+        ycombinatorlink = 'https://news.ycombinator.com/'
```

Comment:
Move this to class level.

Reply:
if i do that error raised that parse requires 5 missing positional arguments

Reply:
That shouldn't be a problem if you do it correctly and use `self`

Reply:
done

Reply:
Not resolved yet.